content/en/releases/platforms.md

```diff
@@ -0,0 +1,336 @@
+addressed. This includes [conformance testing][2] as use of the Kubernetes
+trademark is controlled through a conformance ensurance process. So we are
+looking for folks here to work with [the conformance sub project][3] in addition
+to testing and release.
+
+[2]: https://github.com/cncf/k8s-conformance
+[3]: https://bit.ly/sig-architecture-conformance
+
+#### Step 4: Finishing
+
+If you got this far, you really have made it! You have a clear engagement with
+the community, you are working seamlessly with all the relevant SIGs, you have
+your content in the Kubernetes release and get end users to adopt your
+architecture. Having achieved conformance, you will gain conditional use of the
+Kubernetes trademark relative to your offerings.
+
+#### Generic rules to consider
+
+- We should keep it easy for contributors to get into Step 1.
+- Step 1, by default things should not build and should be switched off.
+- Step 1, should not place undue burden on review or infrastructure (case in
+  point - Windows).
+- Once Step 2 is done, we could consider switching things on by default (but
+  still not in release artifacts).
+- Once Step 3 is done, binaries / images in arch can ship with release.
+- Step 2 is at least the default e2e-gce equivalent, PLUS the node e2e tests.
+  More the better.
+- Step 2 will involve 3rd party reporting to test-grid at the least.
+- Step 2 may end up needing boskos etc to run against clouds (with these arches)
+  where we have credits.
+- Step 3 is at least the conformance test suite. More the better. Using
+  community tools like prow/kubeadm is encouraged but not mandated.
+- Step 4 is where we take this up to CNCF trademark program. For at least a year
+  in Step 3 before we go to Step 4.
+- If at any stage things bit rot, we go back to a previous step, giving an
+  opportunity for the community to step up.
+
+### Deprecating and removing supported platforms
+
+Supported platforms may be considered as deprecated for various reasons, for
+example if they are being replaced by new ones, are not actively used or
+maintained any more. Deprecating an already supported platform has to follow a
+couple of steps:
+
+1. The platform deprecation has been announced on k-dev and links to a k/k issue
+   for further discussions and consensus.
+
+1. The deprecation will be active immediately after consensus has been reached
+   at a set deadline. This incorporates approval from SIG Release and
+   Architecture.
+
+1. Removing the supported platform will be done in the beginning of the next
+   minor (v1.N+1.0) release cycle, which means to:
+   - Update the k/k build scripts to exclude the platform from all targets
+   - Update the k/sig-release repository to reflect the current set of supported
+     platforms.
+
+Please note that actively supported release branches are not affected by the
+removal. This ensures compatibility with existing artifact consumers.
```

Comment:
I would say with "string attached", i.e. that this is on the best effort basis. If Go breaks for example, there's nothing we can do, but we'll do everything in our power to accomplish this.

Reply:
I rephrased that to include the best effort note.